### ISSUE TITLE
 Unable to load Font

### ISSUE THREAD

**Issue by danieluisilva** [xhtml2pdf/xhtml2pdf#396]
Hi, 
I'm trying load a font in pdf, but the pdf is always blank.

This is my test html 

```
<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <style type="text/css">
        @media print {
            @font-face {
                font-family: DejaMono;
                src: url("fonts/DejaVuSansMono.ttf");
            }
        }
        .dejamono {
            font-family: DejaMono;
        }
    </style>
</head>
<body>
<span class="dejamono">Lorem Ipsum is simply dummy text of the printing and typesetting industry.</span>
</body>
</html>
```

In xhtml2pdf debug have a information that font is loaded.

Any idea to resolve this problem?

Tks 
DS

**Comment by luisza** [comment]
How did you load the font? Did you use link_callback to locale the font on disk? 
http://xhtml2pdf.readthedocs.io/en/latest/usage.html#using-xhtml2pdf-in-django

**Comment by danieluisilva** [comment]
I load font with the full path to font.

The weird thing is that in debug info show:

`DEBUG [xhtml2pdf] C:\X\X\lib\site-packages\xhtml2pdf\context.py line 853: Load font u'C:\\X|\X\X\\fonts\\DejaVuSansMono.ttf'`

**Comment by danieluisilva** [comment]
Hi, 

Any solution or workaround?

**Comment by SimonAstani** [comment]
You are not providing complete path of the font location.
Try providing complete path. 
eg if you are in linux .
          @font-face {
                font-family: DejaMono;
                src: url("/home/divyatech/Desktop/fonts/DejaVuSansMono.ttf");
            }

The better solution is to add
 context_dict['STATIC_ROOT'] = settings.STATIC_ROOT
and use like 
        @font-face {
            font-family: 'Noto Sans', sans-serif;
            src : url("{{STATIC_ROOT}}/fonts/NotoSans-Bold.ttf");
        }

in your templates

**Comment by luisza** [comment]
This was fixed on #505
